sdk/storage/azure_storage_blob/src/clients/blob_service_client.rs

```diff
@@ -98,6 +99,31 @@ impl BlobServiceClient {
         self.client.list_containers_segment(options)
     }
 
+    /// Returns a list of blobs across all containers whose tags match a given search expression.
+    ///
+    /// # Arguments
+    ///
+    /// * `filter_expression` - The expression to find blobs whose tags matches the specified condition.
+    ///   eg.
+    /// ```text
+    /// "\"yourtagname\"='firsttag' and \"yourtagname2\"='secondtag'"
+    /// ```
+    ///   To specify a container, eg.
+    /// ```text
+    /// "@container='containerName' and \"Name\"='C'"
+    /// ```
+    ///   See [`format_filter_expression()`](crate::format_filter_expression) for help with the expected String format.
```

Comment:
Here too. I'll not repeat for brevity, but please check other members.

Reply:
Good catch, will fix these instances.